### DISCUSSION TITLE
[Feature Request] Comprehensive list of keyboard shortcuts for Cinnamon

### DISCUSSION THREAD

**Discussion by spamegg1** [linuxmint/discussions#496] [Ideas]
Some users have been asking for it in the Forum.
https://forums.linuxmint.com/viewtopic.php?p=2482362

I think the Shortcuts tab in the Keyboard app in the Menu is good enough, but I guess people need a "list" of some sort that they can search through more easily.

Users keep forgetting shortcuts, so it would help:
https://forums.linuxmint.com/viewtopic.php?p=2482356#p2482356

**Comment by mtwebster** [comment]
In Cinnamon 6.0, you could search for shortcuts (in that keyboard tab).

In 6.2, you'll also be able to reverse search (trigger a shortcut to bring up its function), and Spice (applet, desklet, extension) shortcuts will be included as well.

This is essentially a searchable list that is automatically updated when you install/remove spices. I don't see the need for anything else.

![image](https://github.com/linuxmint/discussions/assets/262776/906706f0-074c-443f-8048-88f2fe31dfde)

**Reply by spamegg1** [reply]
I'll post this information in the forums from now on. Thanks!
